lti_consumer/static/css/student.css

```diff
@@ -0,0 +1,41 @@
+.xblock-student_view.xblock-student_view-lti_consumer h2.problem-header {
```

Comment:
@douglashall Good point.  I'm not sure whether this would be a good idea either, so let's just keep this file here for now.

One thing we could do to prevent this file from getting out of sync with the SCSS source is to add a test to the Travis build that checks whether the file is still current.  Something like this might do:
```yaml
addons:
  apt:
    packages:
      - ruby-sass
script:
  - sass lti_consumer/static/sass/student.scss | cmp - lti_consumer/static/css/student.css
```
Apart from this being completely untested, it might also unreasonably extent build times or be unpractical for another reason.  If you decide trying something like this, I'd be interested in how it goes. :)